13.technical-support/01.How-to-collect-HAR-file/docs.en.md

```diff
@@ -0,0 +1,59 @@
+---
+title: How to collect HAR file
+published: true
+taxonomy:
+    category:
+        - docs
+---
+
+HAR is short for "HTTP Archive", which is a file format for logging of a web browser's interaction with a site.
+
+
+* [Google Chrome](#Chrome)
+* [Firefox](#Firefox)
+* [Edge/Internet Explorer](#Explorer)
+
+
+<a id="Chrome"></a>
+## Google Chrome
+
+You can enable HTTP session recording on the Network (Network) tab in the Developer Tools section of Chrome.
+
+Select “Developer Tools” in the menu:
+
+ **Menu - Advanced Tools - Developer Tools, or press Ctrl + Shift + C on the keyboard**
+
+Click the Network tab.
+Find the round button in the upper left corner of the tab. It should be red. If it is gray, click it to start recording.
+Set the Save Log box (Preserve Log).
+To remove unnecessary data in the header, click on the crossed out circle icon( Clear button). 
+Reproduce the error.
+Save as HAR table with contents click on RMB (Save as HAR file with data)
```

Comment:
Save the network log into a HAR file by right-clicking on any entry in the log and selecting "Save all as HAR with content".